lib/inferno/apps/web/serializers/test_group.rb

```diff
@@ -10,11 +10,17 @@ class TestGroup < Serializer
         field :description
         field :short_description
         field :input_instructions
-        field :test_count
         field :run_as_group?, name: :run_as_group
         field :user_runnable?, name: :user_runnable
         field :optional?, name: :optional
 
+        field :test_count do |group, options|
+          suite_options = options[:suite_options]
+          child_test_count = group.tests(suite_options).count
+          group.groups(suite_options).inject(child_test_count) do |total_test_count, group|
```

Comment:
You shouldn't have to do this in a serializer. A runnable should be able to iterate over its children and figure out its own test count.

Reply:
I see what you mean, will update.